test/modules/generic-mqtt-tester/docker/arm64v8/base/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM arm64v8/ubuntu:18.04
+
+RUN apt-get update && apt-get install -y libssl-dev 
+
+# Add an unprivileged user account for running the module
+RUN useradd -ms /bin/bash moduleuser
+USER moduleuser
```

Comment:
We set USER in base and in module's Dockerfile. Not really a problem, but not necessary.